### DISCUSSION TITLE
Unable to install Windows 7 64-bit on M2 MacBook Air - problem with autounattend.xml.  Secure boot?

### DISCUSSION THREAD

**Discussion by tgkreutz** [utmapp/UTM#5823] [UTM Help (macOS)] [answered]
I'm struggling to install Windows 7 w/SP1 64-bit on a M2 MacBook Air (8 core CPU, 24 GB RAM, macOS Sonoma 14.0) on UTM 4.4.4 (92).  I'm using template "Windows 7.utm" from https://mac.getutm.app/gallery/windows-7 and following the helpful video https://www.youtube.com/watch?v=Tp3REPE1WsQ.  ﻿﻿I've specified Architecture=x86_64 and System="Standard PC (035 + ICH9, 2009) (alias of pc-q35-7.2) (q35)", and have given it plenty of RAM, disk space and cores.
![image](https://github.com/utmapp/UTM/assets/23145742/410bd63e-c3fe-458e-ab5d-cb38df5569fa)
   Initially I was stopped at startup.nsh:
![image](https://github.com/utmapp/UTM/assets/23145742/40ff06db-ceb1-4b11-aba0-28c47d0739e9)
but after reading "Utm stuck on startup.nsh" #3470 (https://github.com/utmapp/UTM/discussions/3470), I deselected the QEMU Tweak "UEFI Boot".  After that, the Windows 7 installation starts properly, with the black "Starting Windows" screen, followed by the blue "Setup is starting..." screen.  Unfortunately, a dialog appears: "Windows could not parse or process unattend answer file. [E:\autounattend.xml] for pass [windowsPE].  The answer file is invalid.":
![image](https://github.com/utmapp/UTM/assets/23145742/055f38ca-408f-44a4-8783-3701c8b5cf21)
  Clicking OK just restarts the entire Windows installation loop and ultimately returns to this dialog box.  This problem exists both for the Windows 7 Ultimate 64-bit iso suggested by https://mac.getutm.app/gallery/windows-7 and also an iso of a Windows 7 Enterprise w/SP1 64-bit DVD (with MAK activation) that I've successfully used to build Windows 7 on many PCs in the past.
  When installing Windows 10, this problem can be surmounted by this procedure: press SHIFT+F10 to open a command prompt, type "regedit", in registry editor navigate to: HKLM/SYSTEM/Setup/Status/ChildCompletion, if setup.exe is 1, change it to 3, restart (https://www.youtube.com/watch?v=KBixbi8ueyE)...but Windows 7 does not have the Status key.  Sigh...
  Some posts suggest that disabling Safe Boot will fix the autounattend.xml problem, but I don't know how to turn off Safe Boot in UTM.
  Thoughts?

**Comment by Leeeeerich** [comment]
Hello! I have the same problems, but when I added new virtual machine, I disabled "install Windows 10 or heigher" param, select emulation on x86_64 and it good setup now!

**Reply by tgkreutz** [reply, marked as answer]
Yes!  This did the trick.  Many thanks.  Starting from scratch (instead of the Windows 7 template from the UTM gallery) was key.  In the Windows dialog box, make sure to disable both "Install Windows 10 or higher" and "UEFI boot".

**Comment by notch8225-commits** [comment]
Just use iOS

**Comment by akashmahedy** [comment]
open settings , uncheck UEFI ,  win7 doesnt support uefi . i also encountered the same problem . this fixed my issue
